TeamViewerPS/Public/Get-TeamViewerUserGroup.ps1

```diff
@@ -0,0 +1,47 @@
+function Get-TeamViewerUserGroup {
+    [CmdletBinding(DefaultParameterSetName = "All")]
+    param(
+        [Parameter(Mandatory = $true)]
+        [securestring]
+        $ApiToken,
+
+        [Parameter(ParameterSetName = "ByUserGroupId")]

```

Comment:
Do you really need `ParameterSetName`, here? Same on the `CmdletBinding`. I think in this case both are not required.